### PR TITLE
Update Tutorial Documentation for 1.0.0 Release

### PR DESCRIPTION
**Pull request by garrettmflynn**
This PR updates the tutorial screenshots in preparation for the 1.0.0 release. A follow-up will address text changes.

Just a note, but I had to bump back the `session_start_time` of the generated test data to get this to pass following https://github.com/NeurodataWithoutBorders/nwb-guide/pull/831 and https://github.com/NeurodataWithoutBorders/nwbinspector/pull/471. Not a huge deal—but it seemed like it was rejecting `datetime` objects indicating the same minute as the specified timezone.

### CODE REVIEW

File: src/pyflask/manageNeuroconv/manage_neuroconv.py [hunk 1]
```diff
@@ -1565,7 +1565,7 @@ def _format_spikeglx_meta_file(bin_file_path: str) -> str:
 
     meta_structure = f"""acqApLfSy=384,384,1
 appVersion=20190327
-fileCreateTime={datetime.now().isoformat(timespec='seconds')}
+fileCreateTime={(datetime.now() - timedelta(hours=1)).isoformat(timespec='seconds')}
```
Comment (CodyCBakerPhD) [inline src/pyflask/manageNeuroconv/manage_neuroconv.py:1568]:
```suggestion
fileCreateTime={(datetime.now() - timedelta(hours=24)).isoformat(timespec='seconds')}
```

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
@garrettmflynn docs build failing

**Comment by CodyCBakerPhD** [comment]
@garrettmflynn Just ping me when you're done updating things

**Comment by garrettmflynn** [comment]
@CodyCBakerPhD Alright should be good to go. Added updates to the tutorial text here as well, since those change the screenshots substantially.

**Pull request merged**